### PR TITLE
refactor: format cpp

### PR DESCRIPTION
**Pull request by sogladev**
## Changes Proposed:
only a style refactor
-  indent switch cases: https://www.azerothcore.org/wiki/cpp-code-standards
- spacing in comments
- long 100+ column `if` statements to `switch`

## Issues Addressed:

- Closes 

## Tests Performed:

- tested ingame (with https://github.com/azerothcore/mod-racial-trait-swap/pull/21), still works
- 

## How to Test the Changes:

1.
2.
3.

### CODE REVIEW

File: src/RacialTraitSwap.cpp [hunk 1]
```diff
@@ -431,7 +429,7 @@ class npc_race_trait_swap : public CreatureScript
 
     bool OnGossipSelect(Player* player, Creature* creature, uint32 /*menuId*/, uint32 gossipListId) override
     {
-        const int32 RTS1 = sConfigMgr->GetOption<uint32>("Racial.Traits.Swap.Gold", 100);
+        int32 const RTS1 = sConfigMgr->GetOption<uint32>("Racial.Traits.Swap.Gold", 100);
```
Comment (sudlud) [inline src/RacialTraitSwap.cpp:432]: Loading an uint into an int 👀

File: src/RacialTraitSwap.cpp [hunk 2]
```diff
@@ -759,328 +780,338 @@ class npc_race_trait_swap : public CreatureScript
...
+                player->learnSpell(20573, false); // Hardiness
+                // Command
+                switch (player->getClass())
+                {
+                    case CLASS_DEATH_KNIGHT: player->learnSpell(54562, false); break;
+                    case CLASS_HUNTER: player->learnSpell(20576, false); break;
+                    case CLASS_SHAMAN: player->learnSpell(65222, false); break;
+                    case CLASS_WARLOCK: player->learnSpell(20575, false); break;
+                    // Command (Non Pet Users)
+                    default: player->learnSpell(21563, false); break;
```
[the remark this answers is not in the document]
Comment (sogladev) [reply src/RacialTraitSwap.cpp:1027]: depends if you want to allow short case labels on a single line 

https://clang.llvm.org/docs/ClangFormatStyleOptions.html#allowshortcaselabelsonasingleline

> The Google C++ Style Guide and the LLVM Coding Standards recommend disabling AllowShortCaseLabelsOnASingleLine

I set it to true to keep the locale switches small

I'll refactor it to disable and let u pick what you prefer

File: src/RacialTraitSwap.cpp [hunk 3]
```diff
@@ -447,311 +464,702 @@ class npc_race_trait_swap : public CreatureScript
...
         {
-            case LOCALE_koKR: localizedExit = LOCALE_EXIT_1; break;
-            case LOCALE_frFR: localizedExit = LOCALE_EXIT_2; break;
-            case LOCALE_deDE: localizedExit = LOCALE_EXIT_3; break;
-            case LOCALE_zhCN: localizedExit = LOCALE_EXIT_4; break;
-            case LOCALE_zhTW: localizedExit = LOCALE_EXIT_5; break;
-            case LOCALE_esES: localizedExit = LOCALE_EXIT_6; break;
-            case LOCALE_esMX: localizedExit = LOCALE_EXIT_7; break;
-            case LOCALE_ruRU: localizedExit = LOCALE_EXIT_8; break;
+            case LOCALE_koKR:
```
Comment (sudlud) [inline src/RacialTraitSwap.cpp:467]: couldn't most of these switches be combined? so that e.g. localizedExit, localizedBack etc. are set within one case

Comment (sogladev) [reply src/RacialTraitSwap.cpp:467]: Didn't notice it lol, they can. will look at it later

**Review by sudlud — APPROVED** [review]

**Pull request merged**